pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -1127,13 +1134,13 @@ def on_back_clicked(self, button):
         # 2) user wants to reclaim some more space => run the ResizeDialog
         # 3) we are just asked to do autopart => check free space and see if we need
         #                                        user to do anything more
-        self.autopart = not self._customPart.get_active() and not self._blivetGuiPart.get_active()
+        self.autopart = self._get_selected_partitioning_method() == PartitioningMethod.AUTO
         disks = [d for d in self.disks if d.name in self.selected_disks]
         dialog = None
         if not self.autopart:
-            if self._customPart.get_active():
+            if self._get_selected_partitioning_method() == PartitioningMethod.CUSTOM:
                 self.skipTo = "CustomPartitioningSpoke"
-            if self._blivetGuiPart.get_active():
+            if self._get_selected_partitioning_method() == PartitioningMethod.BLIVET_GUI:
                 self.skipTo = "BlivetGuiSpoke"
```

Comment:
I think that `blivet_gui.py` needs to be changed as well. Imports from `blivegui` should be moved to the `initialize` method of `BlivetGuiSpoke`, because all spokes are imported dynamically by the hub at the beginning of the installation.

Have you tried to run tests with removed `blivet-gui`? Pylint doesn't complain about it?

Reply:
As we discussed, I'll disable the check on rhel branch.

Reply:
Updated https://github.com/rhinstaller/anaconda/pull/1557